go.sum

```diff
@@ -631,10 +631,11 @@ github.com/subosito/gotenv v1.4.2/go.mod h1:ayKnFf/c6rvx/2iiLrJUk1e6plDbT3edrFNG
 github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7 h1:epCh84lMvA70Z7CTTCmYQn2CKbY8j86K7/FAIr141uY=
 github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7/go.mod h1:q4W45IWZaF22tdD+VEXcAWRA037jwmWEB5VWYORlTpc=
 github.com/tarm/serial v0.0.0-20180830185346-98f6abe2eb07/go.mod h1:kDXzergiv9cbyO7IOYJZWg1U88JhDg3PB6klq9Hg2pA=
```

Comment:
i don't understand why you routinely packed deps update in PRs.
this really is not a clean way to update deps. it defeats the purpose of depedantbot

Reply:
I have (most) spacemesh repositories checked out locally in the same workspace. If one of the dependencies of go-spacemesh updates its `go.mod` file `go work sync` will also update the dependencies here. I'll try to avoid this but it makes changes across repositories much easier 🙂